resource-adapters/api/src/main/java/org/wildfly/swarm/resourceadapters/IronJacamarXmlAssetImpl.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.resourceadapters;
+
+import java.io.IOException;
+import java.io.StringWriter;
+
+import org.wildfly.swarm.config.resource.adapters.ResourceAdapter;
+import org.wildfly.swarm.config.resource.adapters.resource_adapter.AdminObjects;
+import org.wildfly.swarm.config.resource.adapters.resource_adapter.ConfigProperties;
+import org.wildfly.swarm.config.resource.adapters.resource_adapter.ConnectionDefinitions;
+import org.wildfly.swarm.container.util.XmlWriter;
+
+/**
+ * @author Ralf Battenfeld
+ */
+enum IronJacamarXmlAssetImpl {
```

Comment:
Yes, there is one. I was actually doing this for the descriptor project. But there is no one for the resource adapter since this is hosted in the ironjacamar project. I see your point. I'd like to enhance this when there is a ironjacamar.xml support in the config api as Heiko mentioned.